src/pages/IndividualHawker/IndividualHawker.module.scss

```diff
@@ -0,0 +1,138 @@
+@import '../../variables.scss';
+.site-content {
+  padding: 0 12%;
+  padding-bottom: 50px;
+  background: #f9f8f4;
+}
+
+.hawker {
+  display: inline-grid;
+  grid-template-columns: auto auto auto;
+  grid-column-gap: 50px;
+  padding: 38px;
```

Comment:
Can use a smaller padding on mobile. The current padding looks a bit too much when switched to mobile view.

![image](https://user-images.githubusercontent.com/5328196/108992621-7c91b400-76d4-11eb-885a-8e010b3d2ba3.png)